app/controllers/data_exports_controller.rb

```diff
@@ -55,6 +78,21 @@ def destroy
     end
   end
 
+  # Delete from individual data_export page
+  def remove
```

Comment:
**nitpick (blocking):** Instead of adding another `destroy` endpoint, can you instead just add a `redirect` param to `destroy` so that it redirects instead of rendering. That way we are not duplicating endpoints.

Reply:
By that I mean in `destroy` you check if `redirect` is specified and then perform redirect instead of render.

Reply:
Fixed in 0cce62cbcf934784a58a3d9264a98792162aa526

Reply:
Forgot to take out remove view a1d163dbda8bb07f37c890fa2be905d6817a1eac